### PR TITLE
Add petition import management command

### PR DESCRIPTION
**Pull request by robhudson**

### CODE REVIEW

**Review by pmac — APPROVED** [review]
Looks good! Let's give it a shot!

File: basket/petition/models.py [hunk 1]
```diff
@@ -18,7 +19,7 @@ class Petition(models.Model):
     referrer = models.TextField(blank=True)
     token = models.UUIDField()
     email_confirmed = models.BooleanField(default=False)
-    created = models.DateTimeField(auto_now_add=True)
+    created = models.DateTimeField(default=timezone.now)
```
Comment (stevejalim) [inline basket/petition/models.py:22]: Nice

**Review by stevejalim — APPROVED** [review]
r+

File: basket/petition/models.py (hunk 1 above)
Comment (robhudson) [reply basket/petition/models.py:22]: I didn't realize the `auto_now_add` would completely ignore a datetime you pass in when creating a new record. So I changed it.

**Pull request merged**